submissions/tracey-kyle/scripts/models/article.js

```diff
@@ -0,0 +1,156 @@
+(function(module) {
+  function Article (opts) {
+    Object.keys(opts).forEach(function(e, index, keys) {
+      this[e] = opts[e];
+    },this);
+  }
+
+  Article.all = [];
+
+  Article.prototype.toHtml = function(scriptTemplateId) {
+    var template = Handlebars.compile($(scriptTemplateId).text());
+
+    this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+    this.publishStatus = this.publishedOn ? 'published ' + this.daysAgo + ' days ago' : '(draft)';
+    this.body = marked(this.body);
+
+    return template(this);
+  };
+
+  Article.createTable = function(callback) {
+    webDB.execute(
+      'CREATE TABLE IF NOT EXISTS articles (' +
+        'id INTEGER PRIMARY KEY, ' +
+        'title VARCHAR(255) NOT NULL, ' +
+        'author VARCHAR(255) NOT NULL, ' +
+        'authorUrl VARCHAR (255), ' +
+        'category VARCHAR(20), ' +
+        'publishedOn DATETIME, ' +
+        'body TEXT NOT NULL);',
+      function() {
+        console.log('Successfully set up the articles table.');
+        if (callback) callback();
+      }
+    );
+  };
+
+  Article.createTable();
```

Comment:
This works, but it's kind of weird to have it just floating around here. Where do we mostly call our functions? What's the string holding everything together? (the controller)